analyzer/src/functTest/java/org/jboss/gm/analyzer/alignment/GrpcLikeLayoutFunctionalTest.java

```diff
@@ -105,4 +108,23 @@ public void ensureAlignmentFileCreatedAndAlignmentTaskRun() throws IOException,
         verify(1, postRequestedFor(urlEqualTo("/da/rest/v-1/reports/lookup/gavs")));
     }
 
+    @Test
+    public void ensureMissingGroupIdThrowsException() throws IOException, URISyntaxException {
+
+        final File projectRoot = tempDir.newFolder("grpc-like-layout");
+
+        //noinspection ConstantConditions
+        org.apache.commons.io.FileUtils.copyDirectory(Paths
+                .get(TestUtils.class.getClassLoader().getResource(projectRoot.getName()).toURI()).toFile(), projectRoot);
+        //noinspection ResultOfMethodCallIgnored
+        new File(projectRoot, "settings.gradle").delete();
+
+        try {
```

Comment:
Perhpas we could use something like: https://github.com/project-ncl/gradle-manipulator/blob/master/common/src/test/java/org/jboss/gm/common/model/ManipulationModelTest.java#L27

to stay consistent?